packages/syft/src/syft/service/user/user_roles.py

```diff
@@ -36,14 +37,14 @@ class ServiceRole(Enum):
     # Disabling it, as both property and classmethod only works for python >= 3.9
     # @property
     @classmethod
-    def roles_descending(cls) -> List[Tuple[int, Self]]:
+    def roles_descending(cls) -> List[Tuple[int, "ServiceRole"]]:
         tuples = []
         for x in cls:
             tuples.append((x.value, x))
         return sorted(tuples, reverse=True)
 
     @staticmethod
-    def roles_for_level(level: Union[int, Self]) -> List[Self]:
+    def roles_for_level(level: Union[int, "ServiceRole"]) -> List["ServiceRole"]:
```

Comment:
This should be a `@classmethod` instead.
Then line 52 `service_roles = ServiceRole.roles_descending()` should be changed to `service_roles = cls.roles_descending()`